### ISSUE TITLE
React Localization 

### ISSUE THREAD

**Issue by k88hudson** [mozilla/remote-newtab#162]
I think it would be preferable to use https://github.com/yahoo/react-intl since it hooks into the browser's native [Intl](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Intl) stuff and isn't too big. Alternatively, we could use a redux store, but I think this is fine.

Past work in https://github.com/mozilla/remote-newtab/issues/114

cc @sarracini

**Comment by sarracini** [comment]
I agree with using something other than a redux store, since the strings never change once they're set and never change 'state', it seems unnecessary to use a store for them. 
The react-intl looks pretty good. Can we cache the strings and then just retrieve them in react's view?

**Comment by k88hudson** [comment]
@sarracini yep, since we have to prerender a page for each locale anyway (see https://github.com/mozilla/remote-newtab/issues/163) we could just render them into the page as a global. We could use fetch it at runtime, but that add a request and complicate the serviceworker configuration a bit
